bin/worker

```diff
@@ -6,9 +6,9 @@ import argparse
 import os
 import sys
 
-from datetime import datetime
-from rq.worker import logger
-from rq.logutils import setup_loghandlers
+from datetime        import datetime
```

Comment:
I can't comment on the JS changes since I don't know what kids do these days, but I'd recommend we adopy `prettify` if we want to go that route.